### DISCUSSION TITLE
Laravel (with backpack) in subfolder

### DISCUSSION THREAD

**Discussion by juniormcmillan** [Laravel-Backpack/community-forum#403] [Q&A (Help)] [answered]
Hi,

I have Laravel 9 and backpack latest ver installed in  a subfolder on my server.

The issue I have is that most of the CSS and Javascript files are looking in the wrong place, not relative to the install location, but to the document root.

Is there anyone that has managed to install laravel to a subfolder and not to the document root, and got this working?

Thanks

**Comment by pxpm** [comment, marked as answer]
@juniormcmillan you are probably missing to configure your server to point to Laravel `/public` folder? 
I think creating a vhost that points to your `instalation_folder/public` folder would solve your issue.

Let me know,
Cheers

**Reply by juniormcmillan** [reply]
Your answer was on the right trac, so I'll make as correct.

I'm creating a blog in laravel, but there is my website in a different Front controller system, so in the end, laravel only played nice if in the root folder for me.

I had to create htaccess rule so that both live together for now. 

# Redirect requests for /blog/ to /public/ (if the URL starts with /blog/)
RewriteRule ^blog/(.*)$ /public/ [QSA,L]
